Eatery/Views/Giving Day/GivingDayView.swift

```diff
@@ -47,7 +47,7 @@ class GivingDayView: UIView {
         addSubview(bottomPortionView)
 
         supportDescriptionTextView = UITextView()
-        //line spacing
+        // line spacing
```

Comment:
You do lowercase first letter in some places but uppercase in others. Keep it consistent with the whole codebase.